Doc/library/stdtypes.rst

```diff
@@ -1549,6 +1549,33 @@ expression support in the :mod:`re` module).
    interpreted as in slice notation.
 
 
+.. method:: str.removeprefix(prefix, /)
+
+   If the string starts with the *prefix* string, return
+   ``string[len(prefix):]``. Otherwise, return a copy of the original
+   string::
+
+      >>> 'TestHook'.removeprefix('Test')
+      'Hook'
+      >>> 'BaseTestCase'.removeprefix('Test')
+      'BaseTestCase'
+
+   .. versionadded:: 3.9
+
+.. method:: str.removesuffix(suffix, /)
+
+   If the string ends with the *suffix* string, return
+   ``string[:len(string)-len(suffix)]``. Otherwise, return a copy of the
```

Comment:
Why not removing len(string)?

```suggestion
   ``string[:-len(suffix)]``. Otherwise, return a copy of the
```

Reply:
Just removing it would create a an issue with ``string[:-0]`` when the ``suffix`` is empty, so I'll change it to
```
   If the string ends with the *suffix* string and the *suffix* is not empty, return
   ``string[:-len(suffix)]``. Otherwise, return a copy of the
```

Reply:
Alright. Adding " and the *suffix* is not empty" sounds reasonable.